scripts/lib/install

```diff
@@ -33,6 +33,17 @@ ZULIP_PATH="$(realpath $(dirname $0)/../..)"
 # setup-apt-repo does an `apt-get update`
 "$ZULIP_PATH"/scripts/lib/setup-apt-repo
 
+case $1 in
```

Comment:
Get `getopt` to read arguments; we'll likely want to add others in the future.  (and we need to read the domain)

Reply:
Apart from the bashism, I had managed to test the automation script on Travis: https://travis-ci.org/rht/zulip/jobs/279140673. Blocked by the tls-sni-01 challenge for an invalid domain name. To put it in another way, this PR is almost done